digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/csvDataImporter/MassTransitStopCsvImporter.scala

```diff
@@ -260,8 +260,9 @@ trait MassTransitStopCsvImporter extends PointAssetCsvImporter {
     }
   }
 
-  def getNearestRoadLink(lon: Double, lat: Double, user: User, roadTypeLimitations: Set[AdministrativeClass]): Seq[RoadLink] = {
-    val closestRoadLinks = roadLinkService.getClosestRoadlinkForCarTraffic(user, Point(lon, lat)).
+  def getNearestRoadLink(lon: Double, lat: Double, user: User, roadTypeLimitations: Set[AdministrativeClass], stopType: BusStopType): Seq[RoadLink] = {
+    val isCarStop = if (stopType == BusStopType.Tram) false else true
```

Comment:
```suggestion
    val isCarStop = if (stopType != BusStopType.Tram) true else false
```

Reply:
```suggestion
    val isCarStop =stopType != BusStopType.Tram
```

tääkin varmaan myös kelpaisi